playwright-e2e/pages/dsm/participantList-page.ts

```diff
@@ -1,9 +1,13 @@
 import { expect, Locator, Page } from '@playwright/test';
-
 import { waitForNoSpinner } from 'utils/test-utils';
 import { Filters } from 'lib/component/dsm/filters/filters';
 import { ParticipantListTable } from 'lib/component/dsm/tables/participantListTable';
 
+export enum SearchFieldLabel {
+  ShortId = 'Short ID',
+  ParticipantID = 'Participant ID'
+}
+
 export default class ParticipantListPage {
```

Comment:
`ParticipantListPage extends DSMPageBase`? because you created new abstract class `DSMPageBase`.